.gitlab-ci.yml

```diff
@@ -195,7 +195,7 @@ test-build-linux-stable:
   script:
     - ./scripts/gitlab/test_linux_stable.sh
     # we're using the bin built here, instead of having a parallel `build-linux-release`
-    - time cargo build --profile testnet --verbose --bin polkadot
+    - time cargo build --profile testnet --features pyroscope --verbose --bin polkadot
```

Comment:
We probably want to disable this for the time being, since we still have panics from the `piroscope` crate

Reply:
enable again?